contrib/docker/Dockerfile.oxend

```diff
@@ -0,0 +1,28 @@
+FROM ubuntu:20.04
+
+ARG USER_ID
+ARG GROUP_ID
+
+COPY ./oxen-linux-x86_64-8.1.6/* /usr/local/bin/
+
+# https://vsupalov.com/docker-shared-permissions/
+# removed due to "addgroup: The GID `100' is already in use"
+# RUN addgroup --gid $GROUP_ID oxen && 
+
+RUN adduser --system --disabled-password --uid $USER_ID --gid $GROUP_ID oxen && \
+    mkdir -p /wallet /home/oxen/.oxen && \
+    chown -R oxen:$GROUP_ID /home/oxen/.oxen && \
+    chown -R oxen:$GROUP_ID /wallet
+
+# Contains the blockchain
+VOLUME /home/oxen/.oxen
+
+EXPOSE 22022
+EXPOSE 22023
+
+# switch to user oxen
+USER oxen
+WORKDIR /home/oxen
+
+ENTRYPOINT ["oxend", "--p2p-bind-ip=0.0.0.0", "--p2p-bind-port=22022", "--rpc-admin=0.0.0.0:22023", "--non-interactive", "--confirm-external-bind"]
```

Comment:
I don't think `--confirm-external-bind` is needed anymore (with the `--rpc-admin` configuration mode, since the option is obvious enough).

That said, is this actually needed at all?  All of the examples I see below run the wallet from within the container, where the default localhost rpc admin should be usable.

Reply:
You're right, it's not needed to just run the wallet from inside the same container. But I have plans to run some other software against it from outside the container. I'll take a look at those startup flags though.